docs/learn/learn-nominator.md

```diff
@@ -34,18 +35,22 @@ generate.
 
 While your DOT are staked by nominating a validator, they are 'locked' (bonded). You can receive new
 DOT in your account but you cannot stake as validator or transfer DOT away from your account. You
-can [un-nominate at any time](../maintain/maintain-guides-how-to-nominate-polkadot.md) to stop staking your
-funds. Keep in mind that the un-nomination is effective in the next era, and that un-nominating does
-not automatically unbond your funds. There is an unbonding period of 7 days on Kusama and 28 days on
-Polkadot before bonded funds can be transferred after issuing an unbond transaction.
+can [un-nominate at any time](../maintain/maintain-guides-how-to-nominate-polkadot.md) to stop
+staking your funds. Keep in mind that the un-nomination is effective in the next era, and that
+un-nominating does not automatically unbond your funds. There is an unbonding period of
+{{ polkadot: 28 days on Polkadot :polkadot }} {{ kusama: 7 days on Kusama :kusama }} before bonded
+funds can be transferred after issuing an unbond transaction. This is calculated by taking the
+`bonding duration` (in eras), multiplying it by the `length of a single era` (in hours), and
+dividing by the `hours in a day` (24). Example:
+({{ polkadot: 28 × 24 ÷ 24 = 28 days :polkadot }}{{ kusama: 28 × 6 ÷ 24 = 7 days :kusama }}).
```

Comment:
This might be overkill but I found it useful for understanding how to derive the value of the unbonding period from on-chain values.  It renders as:

![image](https://user-images.githubusercontent.com/13341935/206012723-2df05d5a-4f99-4bf5-b20c-293664994354.png)

Reply:
Certainly improves the reader's experience! Thanks